ui/app/mainui/Popups.qml

```diff
@@ -397,6 +407,43 @@ QtObject {
             }
         },
 
+        Component {
+            id: markAsIDVerifiedPopupComponent
+            MarkAsIDVerifiedDialog {
+                onAccepted: {
+                    rootStore.contactStore.verifiedTrusted(publicKey)
+                    Global.displaySuccessToastMessage(qsTr("%1 ID verified").arg(mainDisplayName))
+                    close()
+                }
+                onClosed: destroy()
+            }
+        },
+
+        Component {
+            id: removeIDVerificationPopupComponent
+            RemoveIDVerificationDialog {
+                onAccepted: {
+                    rootStore.contactStore.removeTrustStatus(publicKey)
+
+                    if (markAsUntrusted && removeContact) {
+                        rootStore.contactStore.markUntrustworthy(publicKey)
+                        rootStore.contactStore.removeContact(publicKey)
+                        Global.displaySuccessToastMessage(qsTr("%1 ID verification removed, removed from contacts and marked as untrusted").arg(mainDisplayName))
```

Comment:
suggestion: assign message to a variable and call `Global.displaySuccessToastMessage` once if this variable is not null

Reply:
I don't see how it could be `null` here? :thinking: